CommonTools/ParticleFlow/python/pfPileUp_cfi.py

```diff
@@ -8,5 +8,39 @@
     # pile-up identification now enabled by default. To be studied for jets
     Enable = cms.bool(True),
     verbose = cms.untracked.bool(False),
-    checkClosestZVertex = cms.bool(True)
+    checkClosestZVertex = cms.bool(True),
+    usePrimaryVertexAssignment = cms.bool(False),
+    assignmentQualityForPrimary = cms.int32(2),
+    Jets = cms.InputTag("ak4PFJetsTmp"),
+    assignment = cms.PSet(
```

Comment:
this looks like a 4th copy-paste of the same set of parameters.
This PSet should be defined in one place.
... also, need to understand with PF if usePrimaryVertexAssignment = True can become meaningful outside of the phase-2 scope proposed here. If not, this PSet is not needed in the default.